eng/pipelines/templates/steps/test.yml

```diff
@@ -90,3 +90,9 @@ steps:
       testRunTitle: "$(OSName) - Browser - Unit Tests - [Node $(NodeTestVersion)]"
     condition: and(always(),eq(variables['TestType'], 'browser'))
     displayName: "Publish browser unit test results"
+
+  - template: /eng/common/pipelines/templates/steps/publish-1es-artifact.yml
+    parameters:
+      ArtifactPath: '$(Build.ArtifactStagingDirectory)/test-proxy.$(jobSuffix).log'
+      ArtifactName: 'test proxy logs $(jobSuffix)'
```

Comment:
This part is what needs to be unique. You might also consider just putting this file into whatever other logs artifact we are uploading, if any.

Reply:
Other artifacts all seems to be from single job so they don't need unique names

Reply:
Looks that in these unit-test jobs we don't produce/upload artifacts.